apps/omg_child_chain/test/omg_child_chain/integration/fee_server_test.exs

```diff
@@ -0,0 +1,155 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.ChildChain.Integration.FeeServer do
```

Comment:
I think the name of the module would need to identify it's indeed a test? (same as happy_path_test.exs and monitor_test.exs)